Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/View/Map/Map.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+import MapKit
+
+struct Map: View {
+    
+    @Binding private var center: CLLocationCoordinate2D
+    @State private var span: MKCoordinateSpan?
+
+    private var region: Binding<MKCoordinateRegion> {
+        Binding {
+            let span = span ?? MKCoordinateSpan(
+                latitudeDelta: 0.2,
+                longitudeDelta: 0.2
+            )
+            
+            return MKCoordinateRegion(center: center, span: span)
+        } set: { region in
+            DispatchQueue.main.async {
```

Comment:
Out of curiosity - do we need to do this setter async? Were there some UI stutters or something along those lines without it?

Reply:
I've got some console message. Running in main thread resolved it.